shopinvader/security/shopinvader_security.xml

```diff
@@ -6,6 +6,10 @@
         <field name="sequence">30</field>
     </record>
 
+    <record id="group_shopinvader_partner_binding" model="res.groups">
+        <field name="name">Shopinvader partner binding</field>
+    </record>
```

Comment:
```suggestion
    <record id="group_shopinvader_partner_binding" model="res.groups">
        <field name="name">Partner Binding</field>
    </record>
```

Reply:
This access right doesn't belong to the shopinvader security group so the name should stay like this.

Reply:
Ah, I see that now.